rewrite-java/src/main/java/org/openrewrite/java/UseStaticImport.java

```diff
@@ -62,21 +61,11 @@ public Validated<Object> validate() {
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         TreeVisitor<?, ExecutionContext> preconditions = new UsesMethod<>(methodPattern);
         if (!methodPattern.contains(" *(")) {
-            int indexSpace = methodPattern.indexOf(' ');
+            int indexSpace = Math.max(methodPattern.indexOf(' '), methodPattern.indexOf('#'));
             int indexBrace = methodPattern.indexOf('(', indexSpace);
             String methodNameMatcher = methodPattern.substring(indexSpace, indexBrace);
             preconditions = Preconditions.and(preconditions,
-                    Preconditions.not(new DeclaresMethod<>("*..* " + methodNameMatcher + "(..)")),
-                    Preconditions.not(new JavaIsoVisitor<ExecutionContext>() {
-                        @Override
-                        public J.Import visitImport(J.Import _import, ExecutionContext ctx) {
-                            if (_import.isStatic() && _import.getQualid().getSimpleName().equals(methodNameMatcher.substring(1))) {
-                                return SearchResult.found(_import);
-                            }
-                            return _import;
-                        }
-                    })
-            );
```

Comment:
This stems from an earlier attempt at avoiding conflicts now better served by `hasConflictingImport`. No need to do this as a precondition as we're not avoiding much work here; it limited us from covering overloaded methods.